### ISSUE TITLE
Cannot build pycurl-7.19.5.1 for armadaxp

### ISSUE THREAD

**Issue by keydon** [SynoCommunity/spksrc#1664]
Hy there,

my spksrc repository is up to date, but i have trouble compiling pycurl-7.19.5.1 for the armadaxp-5.1 architecture.
i get the following error:
![pycurl](https://cloud.githubusercontent.com/assets/2421541/7359147/7b9ffa08-ed3b-11e4-8f74-038064a93201.png)

As far as i can tell the dependencies (curl-7.41.0, openssl-1.0.1m, zlib-1.2.8) are building fine :/
Any ideas whats going wrong here?

Thank you!
keydon

**Comment by Diaoul** [comment]
Pycurl is meant to be compiled as a dependency of the python SPK, not alone.

**Comment by keydon** [comment]
hmm, the reason i tried to compile pycurl is to be used within pyload.
Because the recent update of the pyload package broke it for me. It refuses to start up.

I investigated and executed "python systemCheck.py". The check complains about the missing pycurl module.
But i now realize my mistake. I performed the check with the wrong python, since pyload ships with its own python ;O
With the correct python used the systemCheck is successfull. So the reason for pyload not starting up must be elsewhere.

Thanks for your help! i will try to get pyload running again on my own. so this issue can be closed.
